src/app/core/config/config.service.spec.ts

```diff
@@ -30,13 +31,15 @@ describe("ConfigService", () => {
   }));
 
   it("should use the default config when none is loaded", fakeAsync(() => {
-    const configBefore = service.getAllConfigs("");
-    entityMapper.load.and.throwError("No config found");
+    const defaultConfig = Object.keys(defaultJsonConfig).map((key) => {
+      defaultJsonConfig[key]._id = key;
```

Comment:
why should there be these _id on each config item? They are not present on the config we usually have so far.

Reply:
This is due to the fact that we set the `_id` field when using `getAllConfigs` [here](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/config/config.service.ts#L61).
In this case I *fake* the side effect of the `getAllConfigs` function

Reply:
ok, that export of config previously already contained the _id properties. nevermind